Mathlib/CategoryTheory/Abelian/InjectiveResolution.lean

```diff
@@ -322,7 +322,7 @@ lemma ofCocomplex_d_0_1 :
     (ofCocomplex Z).d 0 1 = d (Injective.ι Z) := by
   simp [ofCocomplex]
 
-#adaptation_note /-- Since nightly-2024-03-11, this takes takes forever now -/
+#adaptation_note /-- Since nightly-2024-03-11, this takes forever now -/
```

Comment:
Also remove the "now"? (Also, possibly out of scope : is this still current?)

Reply:
I haven't checked